app/src/analytics/analytics-client.ts

```diff
@@ -0,0 +1,91 @@
+import crypto from 'crypto';
+
+interface AnalyticsAttributes {
+    [key: string]: any;
+}
+
+interface EventPayload {
+    eventName: string,
+    action: string,
+    actionSubject: string,
+    attributes?: AnalyticsAttributes
+}
+
+enum EnvType {
```

Comment:
We want to avoid using enums in ts 

More info (here)[https://dev.to/ivanzm123/dont-use-enums-in-typescript-they-are-very-dangerous-57bh#:~:text=Excessive%20use%20of%20regular%20ENUMs,scalability%20issues%2C%20and%20maintainability%20issues]

Reply:
:shakes-fist: enums

Im going to leave this after all, mainly because this is used on the front end analytics client, so keeping them the same for now, when we migrate over to single analytics client i will convert!!!!